migration/testdata/v1_JobLogger.post

```diff
@@ -0,0 +1,3 @@
+/ffs/joblogger/ID1/QmPewMLNZEgnLxaenjo9Q5qwQwW3zHZ7Ac973UmeJ6VWHE/1602952162298722885,{"Cid":{"/":"QmPewMLNZEgnLxaenjo9Q5qwQwW3zHZ7Ac973UmeJ6VWHE"},"RetrievalID":"","Timestamp":1602952162298722885,"Jid":"ad6da2a0-5465-4275-a330-2537062765c8","Msg":"Deal 763168 with miner f022142 is active on-chain"}
```

Comment:
`pre` and `post` files are CSV values of `(key,value)`.

Most of these cases are real that I dumped from some hosted powergates and in some cases tuned a bit more for interesting cases or easier IIDs.